.gitignore

```diff
@@ -1,3 +1,5 @@
 /target
 **/*.rs.bk
-Cargo.lock
\ No newline at end of file
+Cargo.lock
+*.vscode
```

Comment:
AFAIK, files like `.vscode` and `.idea` are usually put in one's global gitignore.

Reply:
Yes, I agree, user's global gitignore is a good place for user-specific IDE stuff